### DISCUSSION TITLE
I cannot login to my account through the browser extension

### DISCUSSION THREAD

**Discussion by lokou14** [dani-garcia/vaultwarden#4110] [Q&A]
I built the Vaultware with the latest docker version and I can log in to my web address where I built it, I cannot log in from the browser extensions even though I choose the self-built options and type my account details.
![Uploading 截圖 2023-11-28 下午3.30.49.png…]()

**Comment by lokou14** [comment]
<img width="1440" alt="截圖 2023-11-28 下午3 30 49" src="https://github.com/dani-garcia/vaultwarden/assets/89333400/44f0a701-97c8-4246-b5dc-aed7c8fad84b">

**Comment by BlackDex** [comment]
You are running a to old version of Vaultwarden.
The web-vault version is still the old notation.

Update to the latest stable version that should solve your issue.

**Reply by lokou14** [reply]
[
<img width="935" alt="截圖 2023-11-28 下午8 35 44" src="https://github.com/dani-garcia/vaultwarden/assets/89333400/8b7959d9-b59d-4133-937f-10e908e254c9">
](url)
<img width="1230" alt="截圖 2023-11-28 下午8 35 52" src="https://github.com/dani-garcia/vaultwarden/assets/89333400/a03e4ced-4519-4f4c-aa6b-b28e8112e28b">
hello, i don't know why i pull your image latest many times but it always stay at 1.23.1 instead of 1.30.1

**Reply by BlackDex** [reply]
I suspect you are using a docker mirror, and that that specific mirror has not updated the `latest` tag to point to the correct container hashes.

I suggest to either use `testing` which hopefully will work, or else you need to point to the specific version `1.30.1`.
There is nothing we can do about this.
